tests/lib/rules/self-closing-comp.js

```diff
@@ -107,73 +107,84 @@ ruleTester.run('self-closing-comp', rule, {
   invalid: [
     {
       code: 'var contentContainer = <div className="content"></div>;',
+      output: 'var contentContainer = <div className="content"/>;',
```

Comment:
I'm not sure I understand, as that change makes the tests fail. It seems that if there is no space before the `>` then after fixing there should continue to be no space before the `>`.

Would you prefer that the replacing value located in [lib/rules/self-closing-comp.js - Line 88](https://github.com/yannickcr/eslint-plugin-react/pull/770/files#diff-0f0a3a3442c0b696947282a7eac6cedfR88) be changed from `"/>"` to `" />"`?